libvmaf/tools/cli_parse.c

```diff
@@ -287,13 +287,29 @@ static void aom_ctc_v1_0(CLISettings *settings, const char *const app)
         parse_feature_config("psnr_hvs", app);
 }
 
+static void aom_ctc_v2_0(CLISettings *settings, const char *app)
+{
+    aom_ctc_v1_0(settings, app);
+}
+
+static void aom_ctc_v3_0(CLISettings *settings, const char *app)
+{
+    aom_ctc_v1_0(settings, app);
+    settings->feature_cfg[settings->feature_cnt++] =
+        parse_feature_config("cambi", app);
+}
+
 static void parse_aom_ctc(CLISettings *settings, const char *const optarg,
                           const char *const app)
 {
     if (!strcmp(optarg, "proposed"))
         usage(app, "`--aom_ctc proposed` is deprecated. Use `--aom_ctc v1.0`");
-    else if (!strcmp(optarg, "v1.0") || !strcmp(optarg, "v2.0"))
+    if (!strcmp(optarg, "v1.0"))
         aom_ctc_v1_0(settings, app);
+    if (!strcmp(optarg, "v2.0"))
+        aom_ctc_v2_0(settings, app);
+    if (!strcmp(optarg, "v3.0"))
+        aom_ctc_v3_0(settings, app);
```

Comment:
wouldn't the removal of the else chain without an early return potentially call `usage()` on `--aom_ctc v1.0`, or am I missing something else?